src/edu/ucsb/nceas/metacat/properties/PropertiesWrapper.java

```diff
@@ -140,16 +140,21 @@ protected String getProperty(String propertyName) throws PropertyNotFoundExcepti
 
         String returnVal = null;
         if (envSecretKeyMappings.containsKey(propertyName)) {
-            returnVal = System.getenv(envSecretKeyMappings.getProperty(propertyName));
+            String envKey = envSecretKeyMappings.getProperty(propertyName);
+            returnVal = System.getenv(envKey);
+            if (returnVal == null || returnVal.trim().isEmpty()) {
+                logMetacat.warn("Didn't find env secret: " + envKey
+                                    + "; falling back to property: " + propertyName);
+            } else {
+                logMetacat.info(
```

Comment:
I'm open to making this "warn" instead, since it might help admins track down auth problems - might be better than just :silently" overriding the properties value? What do you think, @taojing2002 and @mbjones ?

Reply:
I think WARN is an indication of a problem, and there isn't a problem here AFAICT -- the env property is being used as intended. So, logging it as INFO means there is a way to see it in the logs if someone were debugging a problem, but its a low enough level that it won't get flagged as something that needs to be examined with every WARNING.  See my slack message on where I think the right levels are for typical message types.